project/Build.scala

```diff
@@ -1681,17 +1681,23 @@ object Build {
 
         val s = streams.value
 
+        /* Exclude files coming from Scala's `library-aux` directory, as they are not
+         * meant to be compiled. They are part of the source jar since Scala 2.13.14.
+         */
+        val excludeFiles =
+          Set("Any.scala", "AnyRef.scala", "Nothing.scala", "Null.scala", "Singleton.scala")
```

Comment:
I'm not so happy about this verbatim list. But I guess it is highly unlikely that it changes.